forge/src/test/java/org/openjdk/skara/forge/gitlab/GitLabIntegrationTests.java

```diff
@@ -27,14 +27,14 @@
 import java.util.Arrays;
 import java.util.Properties;
```

Comment:
No longer need "import java.util.Properties"

Reply:
Thanks, fixed in latest!